src/bundle/Resources/richtext/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -361,6 +370,13 @@
           <xsl:value-of select="@class"/>
         </xsl:attribute>
       </xsl:if>
+      <xsl:if test="@style">
+        <xsl:attribute name="mark">
+          <xsl:if test="@style = 'list-style-type:disc;'">disc</xsl:if>
+          <xsl:if test="@style = 'list-style-type:circle;'">circle</xsl:if>
+          <xsl:if test="@style = 'list-style-type:square;'">square</xsl:if>
+        </xsl:attribute>
+      </xsl:if>
```

Comment:
Following the above comment: the value here, coming from `xhtml5/edit` format, indeed might not be trusted, but if you really really need this, I think it would be easier to do so on PHP level prior calling XSLT converter. However for me it's rather not necessary.

I'm fine with KISS here:
```suggestion
      <xsl:if test="contains( @style, 'list-style-type:' )">
        <xsl:attribute name="mark">
          <xsl:call-template name="extractStyleValue">
            <xsl:with-param name="style" select="@style"/>
            <xsl:with-param name="property" select="'list-style-type'"/>
          </xsl:call-template>
        </xsl:attribute>
      </xsl:if>
```

Reply:
Fixed in f7d30bfbc903411207f2739be293b5af84fd4a8a